### PR TITLE
mgr/dashboard_v2: add mirroring page

### PR DESCRIPTION
**Pull request by tspmelo**
This PR uses the api provided by #116. 

localhost_4200_ fh 3

`Signed-off-by: Tiago Melo <tmelo@suse.com>`

### CODE REVIEW

File: src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/block/mirroring/mirroring.component.ts [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+      { prop: 'name', name: 'Image' },
+      { prop: 'description', name: 'Description' },
+      {
+        prop: 'state',
+        name: 'State',
+        cellTemplate: this.stateTmpl
+      }
+    ];
+
+    setTimeout(() => {
```
Comment (votdev) [inline src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/block/mirroring/mirroring.component.ts]: This should be
```
this.interval = setTimeout(() => {
    this.refresh();
}, 30000);
```

**Review by votdev — CHANGES_REQUESTED** [review]

File: src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/block/mirroring/mirroring.component.ts [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        name: 'Health',
+        cellTemplate: this.healthTmpl
+      }
+    ];
+
+    this.pools.columns = [
+      { prop: 'name', name: 'Name' },
+      { prop: 'mirror_mode', name: 'Mode' },
+      { prop: 'leader_id', name: 'Leader' },
+      { prop: 'image_local_count', name: '# Local' },
```
Comment (ricardoasmarques) [inline src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/block/mirroring/mirroring.component.ts]: Number fields should be align to the right.

**Review by LenzGr — DISMISSED** [review]

**Comment by tspmelo** [comment]
Preview, now with data and new column sizes:
192 168 1 136_4200_ fhd 8

**Review by votdev — DISMISSED** [review]

**Comment by tspmelo** [comment]
I added the error/warning information on the menu:
192 168 1 136_4200_ fhd 10

**Review by Devp00l — APPROVED** [review]
lgtm

**Review by LenzGr — APPROVED** [review]

**Pull request merged**